### PR TITLE
Feature/auto login

### PR DESCRIPTION
**Pull request by lacf95**
When a user sign up, he's redirected to the main page, not log in required

### CODE REVIEW

File: app/controllers/registration_controller.rb [hunk 1]
```diff
@@ -5,9 +5,10 @@ def index
 
   def create
     @user = User.new(user_params)
-    if @user.save
-      redirect_to home_index_path
-    end
+    return render 'index' unless @user.save
+    credential = Credential.new(email: @user.email, password: @user.password)
+    session[:user_id] = credential.user[:id] if credential.in?
```
Comment (cesargomez89) [inline app/controllers/registration_controller.rb:10]: what do you mean with `in`? I think you have to change that method name

Comment (cesargomez89) [inline app/controllers/registration_controller.rb:10]: use `credential.user.id` it is not a hash

File: app/controllers/registration_controller.rb [hunk 2]
```diff
@@ -5,9 +5,10 @@ def index
 
   def create
     @user = User.new(user_params)
-    if @user.save
-      redirect_to home_index_path
-    end
+    return render 'index' unless @user.save
+    credential = Credential.new(email: @user.email, password: @user.password)
```
Comment (cesargomez89) [inline app/controllers/registration_controller.rb:9]: what's the point with credential if has_secure_password is doing the job?
also if the user is new you don't need to validate the password with authenticate

**Review by cesargomez89 — CHANGES_REQUESTED** [review]
I left some comments

File: app/controllers/registration_controller.rb (hunk 2 above)
Comment (lacf95) [reply app/controllers/registration_controller.rb:9]: Ok, let me fix all those errors, indeed not need of validation

**Pull request closed without merging**